source/material_model/rheology/visco_plastic.cc

```diff
@@ -617,7 +613,7 @@ namespace aspect
         friction_models.initialize_simulator (this->get_simulator());
         friction_models.parse_parameters(prm);
 
-        use_elasticity = prm.get_bool ("Include viscoelasticity");
+        use_elasticity = this->get_parameters().enable_elasticity;
```

Comment:
Can you either eliminate the whole variable and replace with calls to this->get_parameters().enable_elasticity (they are cheap), or rename the variable to `enable_elasticity`?